### PR TITLE
Add 2 new imgui opcodes

### PR DESCRIPTION
**Pull request by user-grinch**

### CODE REVIEW

File: gta3/gta3.json [hunk 1]
```diff
@@ -29879,6 +29879,35 @@
...
+            },
+            {
+              "name": "height",
+              "type": "float",
+              "source": "var_any"
+            }
+          ],
+          "id": "0C1B",
+          "name": "IMGUI_GET_DISPLAY_SIZE",
+          "num_params": 0,
```
Comment (x87) [inline gta3/gta3.json]: num_params is the sum of input and output params. should be 2

Comment (user-grinch) [reply gta3/gta3.json]: opps, fixing..

Comment (x87) [reply gta3/gta3.json]: please bump the version in the meta section and in version.txt. also update timestamp if you can


if you edit opcodes via the web editor you can add the command once and then clone it to other games, all changes are synchronized.

**Comment by user-grinch** [comment]
Added it using the editor, closing this

**Pull request closed without merging**